src/keeper/contracts/ContractBase.ts

```diff
@@ -119,12 +119,10 @@ export abstract class ContractBase extends Instantiable {
 
             if (value) gas += 21500
 
-            const chainId = await this.web3.eth.net.getId()
             const receipt = await tx.send({
                 from,
                 value,
-                gas,
-                chainId
```

Comment:
`chainId` is required to send transactions to nodes with EIP-155 protection which includes mumbai

Reply:
Initially we had this on mint that it errored out because it complained of a `unknown parameter chainId` on Rinkeby.

Reply:
Adding it back now.